ddtrace/tracer/option.go

```diff
@@ -667,8 +667,8 @@ func WithDogstatsdAddress(addr string) StartOption {
 	}
 }
 
-// WithSamplingRules specifies the sampling rates to apply to spans based on the
-// provided rules.
+// WithSamplingRules specifies the sampling rates to apply to trace spans based on the
+// provided rules. Both trace sampling and single span sampling rules can be configured with this option.
```

Comment:
Same here. There is a new `Type` field now in `SamplingRule`, and that will be announced in a changelog. There is no need to mention it here too, it's a detail of `SamplingRule`.